lisa/spine.sh

```diff
@@ -0,0 +1,200 @@
+#!/bin/bash
+# Written by Lisa Eunyoung Lee - Jun 2022
+# Revised by Lisa Eunyoung Lee - Jan 2023
+# Spinal Cord Toolbox - Version 5.7
+
+# T2 - Spinal Cord CSA averaged between C2-C4
+# MT - Spinal Cord MTR averaged between C2-C4
+
```

Comment:
This script should contain a more complete description so that we remember exactly what it does and how to use it. You can also add links to related issue (ex #88) to bring more context.